dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleStatsConnector.java

```diff
@@ -37,12 +38,31 @@ public OracleStatsConnector() {
   @Override
   public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments) throws Exception {
     StatsTaskListGenerator taskListGenerator = new StatsTaskListGenerator();
-    out.addAll(taskListGenerator.createTasks(arguments));
+    int queryLogDays = getQueryLogDays(arguments);
+    out.addAll(taskListGenerator.createTasks(arguments, queryLogDays));
   }
 
   @Nonnull
   @Override
   public String summary(String fileName) {
     return String.format("Oracle statistics saved to %s", fileName);
   }
+
+  static int getQueryLogDays(ConnectorArguments arguments) {
+    Integer asObject = arguments.getQueryLogDays();
```

Comment:
I would suggest renaming `asObject` to `queryLogDays` to increase readability.

Reply:
Resolving as outdated - the set of correct values was extended and after the rewrite this variable is gone.